### ISSUE TITLE
initialize charts with overal summary on all environments

### ISSUE THREAD

**Issue by mwasilew** [Linaro/squad#198]
Imported from https://projects.linaro.org/browse/QA-1545
Currently the charts page starts empty. Instead, we should display by default the overall summary on all environments.

**Comment by terceiro** [comment]
Yet another failure is properly describing what each issue actually means. :-)

"overall summary" is the geomean of all metrics for a given build/environment. You will be able to get that from the `metrics_summary` field in `Status` objects, with `suite=None`. the data set should be given by something like

```
env = ... # the environment you want
project = ... # the project you want
Status.objects.filter(suite=None, test_run__environment=env, test_run__build__project=project)

```

With that said, you could reword the commit message (dropping mention to #198) and the pull request title and we could get this merged right away and change to the overall summary later. after all showing the test pass % by default is better than displaying nothing.

**Comment by terceiro** [comment]
Just remove the issue # from the commit message so the issue doesnt get closed.
 The link from the PR will stay there forever, its ok

**Comment by chaws** [comment]
@terceiro in your comment above, you mentioned that the `metrics_summary` of `Status` when `suite=None` represents the metrics summary (geomean) of all metrics for a given Build/Environment.

I've run this query but I'm still getting multiple `metrics_summary` per Build/Environment. Then it occurred to me that because a single build has multiple `TestRun`s, when `suite=None` it actually represents metrics summary for a single TestRun/Environment, not for a Build/Environment.

And because `ProjectStatus` contains `metric_summary` of a build (of all environments), we still don't have a simple way to get the overall metrics summary per build per environment.

Am I missing anything here?

**Comment by terceiro** [comment]
@chaws yes, I think you got it right

this part was designed very early and ended up a bit outdated with regards to the whole Build/TestRun design. We indeed don't have this data per build and environment

**Comment by chaws** [comment]
This was already solved in #537
